### DISCUSSION TITLE
Python *.py cells: is it possible to see results inline?

### DISCUSSION THREAD

**Discussion by AlexeyDmitriev** [microsoft/vscode-jupyter#15545] [Questions and Answers] [answered]
So, I like how ipynb's are rendered in vscode.

However, storing a `ipynb` in repository is not as convenient:
* You should check that big outputs are not accidentally stored
* Linter tools that works with python files don't generally work with ipynb (unless they made to do so).

So, storing files with `# %%` in `*.py` ([docs](https://code.visualstudio.com/docs/python/jupyter-support-py)) files seems like a good alternative on the first glance
However, the UI is different: output are stored not between the cells, but on the separate window, it's harder to see how outputs correspond to each other.

Is there option to show `*.py` files closer to how `ipynb`'s are shown?

**Comment by DonJayamanne** [comment, marked as answer]
Duplicate of https://github.com/microsoft/vscode-jupyter/issues/1240
Please do upvote that issue on the top, as thats how we prioritize issues.
